temporal/api/cloud/auditlog/v1/message.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package temporal.api.cloud.auditlog.v1;
+
+option go_package = "go.temporal.io/api/cloud/auditlog/v1;auditlog";
+option java_package = "io.temporal.api.cloud.auditlog.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::AuditLog::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.AuditLog.V1";
+
+import "google/protobuf/timestamp.proto";
+
+// LogRecord represents an audit log entry from Temporal, structured for easy parsing and analysis.
+message LogRecord {
+  // Time when the log was emitted from the source
+  google.protobuf.Timestamp emit_time = 1;
+
+  // Level of the log, i.e. info, warn, error etc
+  LogLevel level = 2;
+
+  // Operator email address
+  string user_email = 3;
+
+  // Operator IP address or server name
```

Comment:
If it can be either, field should not be suffixed with `ip_address`. Maybe a more generic `host` or `address`?

Reply:
Uh I'm actually not entirely sure of this one just copied over the proto. We use x-forwarded-for to populate this field, I think that's IP only?

Reply:
Agree, we should use `host` (preferred) or `address`. @shivam-ajmera we should have ability to do some transformation of field names before exposing it via API